pkgs/development/libraries/science/math/mkl/default.nix

```
@@ -1,4 +1,11 @@
-{ stdenvNoCC, fetchurl, rpmextract, undmg, darwin, enableStatic ? false }:
+{ stdenvNoCC
+, fetchurl
+, pkgconfig
+, rpmextract
+, undmg
+, darwin
+, enableStatic ? false }:
```

```suggestion
, enableStatic ? false
}:
```
Nitpick, the formatters usually put the opening `}:` on its own line, so this can all be sorted and new elements added with a 1-line diff.

---

Thanks, fixed!